### PR TITLE
Add Sucuri

### PR DESCRIPTION
**Pull request by joshtch**

### CODE REVIEW

**Comment by Carlgo11** [comment]
The test seems to be wrong here as the regions array looks fine.

File: img/s/sucuri.net.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...H+p5c9/wkAvIXy7MwmQKScueu1Go92uX8PZ+m633dhtNrvrveJ+u7nbpzLudhv7BUJzyRMMzcygZRVPeWvoAp3VB5j8gK38qg9QiedTSTd3Vt+p1+vrvenkWWMDX+5s1HcPSzx8Zcx1/b6LCZkQbjA0t5mtlGr5w8PV1vLHk0UUUgBn4RwgGa8map6XLMHFYadz9j6ercCvh2ennbPDlbmJ2uGb6zcX4MELfD7KgKs0k2qJZrubilWhB6ecrGIgUrlywmPrKlGjt/Gk2abgr6eox3GexPjzhCghSibVksz3YK5QLSeT5SoGOJXCD8sEty8v4eEO9GrpNN3vaBfi1QP3Cm4WnBsVGtQiHZzM0GiGXLBl+gsO3H3yViDvonLBmtcv2ihsQAfdjXmWB1UbZP6tzbiZs9VNWztyF5XkYCQF5OlpMkRptt5y+Y3RQpb3DTFGtdiRbkLrbpjfNAM3bJoZmdaV7oayYxrRJll9iTXvz9IM7Y5kM0OBUXMYze5H4Tv/kBujWrFmNsgOhuaXUf3Or27m/PIl73XaLZdpptCJo3UK5xKagwd/H7R+T1EVHIKelGhx54+mTn23CmsOF1AZTWZMT08KtNUNSt34mmWjMJuJHKCpgkpzcMu15Fn6LI0OZh3PM9fM0TbNVEAhmppbS13IZxMcaGZoSZaaVacMT52ejYjP6AcZItH2PJvf2uwtaCVPgg428jyW5mBdmT4z1QItpyibzzwbgzolnDqZDZY5lg0CoyHmRAomv+1bW6iCoe7maI7l6HDo2BDlgQ6iIb7RY+7G1Yz3gMisGz2fh3W3Eo0rUxdNseN5HtQp2qhjFRzkM2omtFFB1SjhbWXzWYLlsFOSGRmi5OGpE+1tNmGkgiSao/WdYvt7pEinjHYD0eEeHH/W2XuQxY6JJjRploEONFvms9oqZIcy2u4GUy3BvAct2VAV1MfoaDLokrXJP0Qzq6Akh7ubZ4MboqMDN2zz+W9kQ6Flp1grGL5vDJt1OJL0o/5FzL5TJFk2t5E6fafwdtHJIKOB5GEVlGiL0UEPmqIZmne3nM82n/VtNaiChmL6j2ETSaVuLM0D0EE2bKmzaeY+B2ZEwKSa2THIZ230M5tVEw7sQfUZfwGqJOYIDxHpFAAAAABJRU5ErkJggg==" fill="none" fill-rule="evenodd"/></svg>
```
Comment (Carlgo11) [inline img/s/sucuri.net.svg]: Please use this icon instead
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 90 90"><defs><linearGradient id="a" x1="-4.85" x2="48.7" y1="83.17" y2="42.15" gradientUnits="userSpaceOnUse"><stop stop-color="#22415d"/><stop offset="1" stop-color="#24ba9e"/></linearGradient></defs><path fill="url(#a)" d="m52.25 49.19 15.88 9.15L45 71.7l-15.87-9.16-15.87 9.16L45 90l39-22.5V30.87ZM6 59.13V22.5L45 0l31.74 18.31-15.87 9.16L45 18.31 21.87 31.66l15.87 9.15"/></svg>
```

File: entries/s/sucuri.net.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "Sucuri": {
+    "domain": "sucuri.net",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://docs.sucuri.net/billing/two-factor-authentication/",
+    "categories": [
+      "domains",
```
Comment (Carlgo11) [inline entries/s/sucuri.net.json]: The domains category is meant for domain registrars. It doesn't look like Sucuri sells domains.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/s/sucuri.net.json (hunk 2 above)
Comment (joshtch) [reply entries/s/sucuri.net.json]: Got it. It does provide domain name servers, but yes, it's not a registrar.

**Comment by hkamran80** [comment]
@Carlgo11
> The test seems to be wrong here as the regions array looks fine.

The script doesn't know how to handle regions prefixed with a `-`. I'll submit a PR to fix it.

**Comment by Carlgo11** [comment]
@hkamran80 Already submitted a PR.

**Comment by hkamran80** [comment]
@Carlgo11 Probably should've checked my notifications before. Thanks for letting me know!

**Review by Carlgo11 — APPROVED** [review]

**Comment by joshtch** [comment]
I'm really impressed by your efficiency in getting this merged. Thanks!

**Pull request merged**